### PR TITLE
Use 'NA' instead of leaving certain columns blank

### LINKED ISSUE #170: Fill empty fields in the output tsv with 'NA'

### PR DESCRIPTION
**Pull request by susannasiebert**
Closes #170

### CODE REVIEW

**Comment by chrisamiller** [comment]
+1  Thanks!

File: tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-22	26936775	26936776	G	T	ENST00000338754	ENSG00000128294	missense	P/H	274	TPST2	HLA-A*02:01	9	6	DLIGKHGGV	DLIGKPGGV	2037.56	1079.22	0.530	SMM	2230.64									11952.0	16231.0	1079.22	2037.56	1102.63	2230.64
-22	38119219	38119220	GA	G	ENST00000406386	ENSG00000100106	FS		219	TRIOBP	HLA-A*02:01	9	1596	SQWKKHWFV	NA	10.0	NA	NA	NetMHC	44.99									NA	10.0	NA	44.99	NA	52.53
-22	50682229	50682230	T	C	ENST00000248846	ENSG00000128159	missense	H/R	220	TUBGCP6	HLA-A*02:01	9	4	SLFGALVRS	SLFGALVHS	722.95	565.08	0.782	SMM	728.0									457.0	728.0	565.08	722.95	589.43	771.67
-22	18644672	18644673	C	T	ENST00000215794	ENSG00000184979	missense	A/V	124	USP18	HLA-A*02:01	9	6	RPLELVYCL	RPLELAYCL	863.83	1804.8	2.089	SMMPMBEC	972.99									9916.0	5180.0	1995.77	972.99	1804.8	863.83
+22	41895790	41895791	C	A	ENST00000216254	ENSG00000100412	missense	A/E	33	ACO2	HLA-A*02:01	9	9	KVEMSHFEP	KVAMSHFEP	23862.0	18095.0	0.758	NetMHC	46463.29	38.5035	35.4301	NA	NA	NA	NA	NA	NA	18095.0	23862.0	7483.59	46463.29	8578.87	47362.0
```
Comment (tmooney) [inline tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv]: Does this change have other effects besides setting `NA` on things?  I'm surprised to see new numbers appearing in the output.

Comment (susannasiebert) [inline tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv]: In this PR I updated the input tsv files for the parser tests which I failed to do in some previous PRs (mainly the cufflinks/bam-readcount PR). The updated input tsv files now contain values for FPKMs and NAs for readcounts which get carried through to the parser output. It was really an oversight to not update the tsvs previously.

**Comment by susannasiebert** [comment]
I just noticed that I had accidentally committed a whole bunch of tmp directories. They are now removed.

File: tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv (hunk 1 above)
Comment (tmooney) [inline tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv]: Cool--as long as it's expected!

**Comment by tmooney** [comment]
+1 🎒

Are those `tmp` directories only in this PR?  If so, it might be nice to rebase them away so they're not forever in the history of this repo!

**Comment by susannasiebert** [comment]
Some of them must've been committed in an older PR and some where from this PR. I rebased the last commit to at least get totally rid of the ones that were new.

**Pull request merged**